lib/dbservice/users/teacher.ex

```diff
@@ -11,7 +11,7 @@ defmodule Dbservice.Users.Teacher do
     field :designation, :string
     field :grade, :string
     field :subject, :string
-    field :uuid, :string
+    field :teacher_id, :string
```

Comment:
Don't remove `uuid` -- We can't guarantee uniqueness of `teacher_id` because maybe we'll set it manually or something. Have both `uuid` (system generated) and `teacher_id`

Reply:
`uuid` is not system generated, system generated `id` is different. This is the ID for a teacher that we give, just like `student_id`. I'm just renaming it for better understanding.
![Uploading Screenshot 2023-11-30 at 4.53.35 PM.png…]()